hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/shell/volume/ListVolumeHandler.java

```diff
@@ -85,11 +85,18 @@ protected void execute(OzoneClient client, OzoneAddress address)
     }
 
     int counter = 0;
+    final StringBuilder res = new StringBuilder("[ ");  // Mark JSON array start
```

Comment:
Quick Question:
Why you can't use the `out` itself? Why you are moving to `StringBuilder`? Earlier the flow was we are just producing json and pushing it to the out stream, now If I catch correct we will be storing it till the end and then push, Any reasons for doing so? Does this approach not increase the memory pressure, storing the entire list till end? Or did I interpret it wrong?

Reply:
Done